example/avro.rb

```diff
@@ -0,0 +1,26 @@
+require "bundler/setup"
+require "glass_octopus"
+
+app = GlassOctopus.build do
+  use GlassOctopus::Middleware::CommonLogger
+  use GlassOctopus::Middleware::AvroParser, "http://schema_registry_url:8081"
+
+  run Proc.new { |ctx|
+    puts "Got message: #{ctx.message.key} => #{ctx.message.value}"
+  }
+end
+
+def array_from_env(key, default:)
+  return default unless ENV.key?(key)
+  ENV.fetch(key).split(",").map(&:strip)
+end
+
+
+GlassOctopus.run(app) do |config|
+  config.adapter :poseidon do |kafka_config|
```

Comment:
Please use the ruby-kafka driver in the examples. Poseidon is deprecated and only supports Kafka 0.8.x which is pretty much not supported anymore. Let's promote a more up to date setup for potential users.